web/styles/portico/portico.css

```diff
@@ -239,10 +239,15 @@ html {
         & a {
             &.highlighted {
                 background-color: hsl(152deg 40% 42%);
+            }
+        }
 
+        & a:focus {
+            &.highlighted {
                 /* Don't show the focus outline for the highlighted page. */
                 outline: 0;
             }
+            outline: 5px auto hsl(153deg 32% 55%);
```

Comment:
This belongs above the nested `&.highlighted` class. Also the `outline` style here should be solid, not auto--auto still renders the buggy blue outline ring on other sidebar elements. Your solution here only removes it on the highlighted row.